lib/views/course_view/pinned_courses_view/pinned_courses_content_view.dart

```diff
@@ -16,18 +16,21 @@ import 'package:gocast_mobile/views/course_view/components/pinned_course_card.da
 class PinnedCoursesContentView extends ConsumerWidget {
   final String title;
   final List<PinnedCourseCard> pinnedCourseCards;
+  final Future<void> Function()? onRefresh;
```

Comment:
This `onRefresh` attribute is not needed and can be removed.

Reply:
fixed